tests/jobsautoreport/test_slack.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+from unittest.mock import MagicMock
+
+from jobsautoreport.slack import SlackReporter
+
+
+def test_send_message():
+    test_data = {
+        "success_rate": "test-success-rate",
+        "number_of_jobs_triggered": 30,
+        "average_jobs_duration": timedelta(minutes=30),
+    }
+    test_channel = "abcdefgh"
+    test_blocks = [
```

Comment:
```suggestion
    expected_blocks = [
```
just to emphasis that is... expected